### PR TITLE
[Mobile]Changing Health Tips messages to be dynamic.

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
#### Summary of Changes (What does this PR do?)

- [x] Health tips messages should change according to the air quality of the place e.g 'moderate', 'good'

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
- Checkout the branch
- Run the app and navigate to the `More insights` page.
- Depending on the air quality, the health recommendations should vary as indicated in this [doc](https://docs.google.com/document/d/13wp_MZyJZdcCMEv4vbbS3ghsR_PUTOIVWLFwdVrzogc/edit)

#### What are the relevant tickets?
[MOB-142](https://airqoteam.atlassian.net/browse/MOB-142)

### CODE REVIEW

File: mobile/lib/utils/pm.dart [hunk 1]
```diff
@@ -13,65 +13,35 @@ List<Recommendation> getHealthRecommendations(
   final recommendations = <Recommendation>[];
```
Comment (NoahNsimbe) [inline mobile/lib/utils/pm.dart:13]: Thanks @BenjaminSsempala. 
Could we utilise this PR to rename Recommendation(s) to HealthTip(s).

**Comment by NoahNsimbe** [comment]
Hi @BenjaminSsempala, I think we might also need to review the logic behind health tips. What are they based on?

- [ ] the day's latest air quality reading.
- [ ] the day's forecast data.
- [ ] the day's historical air quality.
- [ ] an ML model for generating the recommendations.

cc @sserurich @Baalmart

**Comment by Baalmart** [comment]
> * [x]  the day's latest air quality reading.
> * [x]  the day's forecast data

Thanks @NoahNsimbe , I would personally consider the first 2 you have shared, that is:

1.  the day's latest air quality reading.
2.  the day's forecast data

**Review by Baalmart — APPROVED** [review]
Thanks @BenjaminSsempala , this LGTM.
After addressing any shared recommendations, this PR should be good to go.

Thanks again!

**Comment by NoahNsimbe** [comment]
> > * [x]   the day's latest air quality reading.
> > * [x]   the day's forecast data
> 
> Thanks @NoahNsimbe , I would personally consider the first 2 you have shared, that is:
> 
> 1. the day's latest air quality reading.
> 2. the day's forecast data

Thanks @Baalmart. We shall incorporate this logic in a succeeding PR. cc @BenjaminSsempala

File: mobile/lib/utils/pm.dart (hunk 1 above)
Comment (BenjaminSsempala) [reply mobile/lib/utils/pm.dart:13]: Thanks @NoahNsimbe 
I've made the changes.

**Review by Mnoble-19 — APPROVED** [review]

**Pull request merged**